mysql-ballerina/Ballerina.toml

```diff
@@ -0,0 +1,28 @@
+[project]
+org-name = "ballerinax"
+version = "@toml.version@"
+
+[platform]
+target = "java8"
+
+    [[platform.libraries]]
+    artifactId = "mysql-native"
+    version = "@project.version@"
+    path = "../mysql-native/build/libs/mysql-native-@project.version@.jar"
+    groupId = "org.ballerinalang"
+    modules = ["mysql"]
+
+    [[platform.libraries]]
+    artifactId = "ballerina-sql"
+    version = "0.5.0"
+    path = "@project.path@/bre/lib/ballerina-sql-0.5.0-java.jar"
+    groupId = "org.ballerinalang"
+    modules = ["mysql"]
```

Comment:
Do we need this now since this is coming from distribution?. once SQL module is moved we need to change anyway

Reply:
Since we are importing java functions from here, I had to add it in the TOML, if not this is throwing class not found. This added even in the ballerina-lang too. Isn't it normal behaviour?

Reply:
Ideally, it should pick from the distribution, since it is already there in bre/lib directory. I also had the same requirement, it worked without adding it in toml file.

Reply:
Raised the issue here, https://ballerina-platform.slack.com/archives/CAXS256RX/p1598615967018500